source/features/one-click-review-submission.tsx

```diff
@@ -91,6 +84,13 @@ function init(): false | void {
 			}
 		});
 	});
+
+	// This will prevent submission when clicking "Comment" and "Request changes" without entering a comment and no other review comments are pending
+	return delegate(form, 'button', 'click', ({delegateTarget: {value}}) => {
```

Comment:
oof, this function should probably be moved out. O think it's only jere because of `form`, which is actually accessible via `delegateTarget.form`